ui/admin/config/ember-cli-update.json

```diff
@@ -3,7 +3,7 @@
   "packages": [
     {
       "name": "ember-cli",
-      "version": "4.4.1",
+      "version": "4.12.2",
```

Comment:
? for learning: does running the `ember-cli-update` cmd, automatically update the package.json dependencies or is that part done manually ?

Reply:
@lisbet-alvarez it will update it automatically, you will have some merge conflicts that you will. have to sort out at the end though.